src/linking_number_certificate.cpp

```diff
@@ -357,15 +357,22 @@ void ComputeLinkingNumbersDirectSum(
  */
 void LinkingNumberCertificate::ComputeFromModel(const Model &model,
                                                 bool force_direct_sum,
+                                                bool discretize,
                                                 double barnes_hut_init_beta,
                                                 double barnes_hut_beta_limit) {
   std::cout << "Starting Potential Link Search." << std::endl;
   std::vector<std::vector<int>> potential_links = PotentialLinkSearch(model);
   std::vector<Curve> discretized_curves;
   std::vector<std::vector<int>> potential_links_unique_per_curve;
+  if (discretize){
```

Comment:
nit: space between ) and {. (use clang formatting -- see the code I pasted in the review comment above)